rust/sealevel/client/src/warp_route.rs

```diff
@@ -544,15 +544,20 @@ fn init_warp_route(
         .map(|s| Pubkey::from_str(s).unwrap())
         .unwrap_or(core_program_ids.mailbox);
 
-    let interchain_gas_paymaster = token_config
-        .connection_client
-        .interchain_gas_paymaster
-        .clone()
-        .map(|config| (config.program_id, config.igp_account))
-        .unwrap_or((
-            core_program_ids.igp_program_id,
-            InterchainGasPaymasterType::OverheadIgp(core_program_ids.overhead_igp_account),
-        ));
+    // TODO for now not specifying an IGP for compatibility with the warp route UI.
```

Comment:
This is intentional - the warp route UI needs to craft specific accounts in order to call into the IGP, which it doesn't support yet. Supporting this will be as simple as:
1. update UI to specify certain accounts when the IGP a warp route is set
2. set the IGP in the warp route